### PR TITLE
Indicate the node where the dump is actually located

### PR DESCRIPTION
**Pull request by jsuchome**
cc @rhafer

### CODE REVIEW

**Comment by rhafer** [comment]
+1 , If this is really what upgrade UI needs.

**Comment by dirkmueller** [comment]
shouldn't that be an override attribute? Just a guess..

**Comment by sayalilunkad** [comment]
probably need to define this somewhere node[:crowbar][:upgrade][:db_dumped_here]. Not sure though.

**Comment by jsuchome** [comment]
@sayalilunkad no, it's not needed (we already define :upgrade submap)
@dirkmueller you mean that we would not need to reset it on the other node? Might be good, but currently I'm glad that this works

File: chef/cookbooks/crowbar/recipes/crowbar-db-dump.rb [hunk 1]
```diff
@@ -68,6 +74,10 @@
     EOF
   end
 
+  # we have to indicate the node where the dump is actually located
+  node[:crowbar][:upgrade][:db_dumped_here] = true
```
Comment (vuntz) [inline chef/cookbooks/crowbar/recipes/crowbar-db-dump.rb]: `node.set` please

**Comment by vuntz** [comment]
+1

**Pull request merged**